mixins/wallet-login.js

```diff
@@ -58,6 +73,37 @@ export default {
         return false;
       }
     },
+    async login(loginPayload = {}) {
+      this.isLoginSuccessful = false;
+      const signer = this.getSigner;
+      const address = this.getAddress;
+      const payload = await signLoginMessage(signer, address);
+      const data = {
+        locale: this.getCurrentLocale,
+        platform: this.platform,
+        ...payload,
+        ...loginPayload,
+      };
+      this.signInPayload = data;
+      try {
+        await this.loginUser(this.signInPayload);
```

Comment:
Why do we need to store `this.signInPayload` instead of just calling `await this.loginUser(signInPayload);`?